src/Artifacts.UnitTests/RobocopyTests.cs

```diff
@@ -181,11 +184,11 @@ public void SingleFileMatchRecursive()
                 Path.Combine("baz", "baz.dll"),
                 Path.Combine("foo", "foo", "foo", "foo.pdb"));
 
-            DirectoryInfo destination = new DirectoryInfo(Path.Combine(TestRootPath, "destination"));
+            var destination = new DirectoryInfo(Path.Combine(TestRootPath, "destination"));
```

Comment:
I hate `var` but its not banned.  Be aware that I'll probably undo this in some future commit...

Reply:
Restored in next push.